docs/analysing-compilation-output.rst

```diff
@@ -0,0 +1,190 @@
+        tag_4:
+          mload(0x40)
+          tag_6
+          swap2
+          swap1
+          tag_7
+          jump	// in
+        tag_6:
+          mload(0x40)
+          dup1
+          swap2
+          sub
+          swap1
+          return
+        tag_5:
+            /* "/tmp/test.sol":50:54  uint */
+          0x00
+            /* "/tmp/test.sol":67:68  1 */
+          0x01
+            /* "/tmp/test.sol":60:68  return 1 */
+          swap1
+          pop
+            /* "/tmp/test.sol":14:72  function one() public pure returns (uint) {... */
+          swap1
+          jump	// out
+            /* "#utility.yul":7:125   */
+        tag_10:
+            /* "#utility.yul":94:118   */
+          tag_12
+            /* "#utility.yul":112:117   */
+          dup2
+            /* "#utility.yul":94:118   */
+          tag_13
+          jump	// in
+        tag_12:
+            /* "#utility.yul":89:92   */
+          dup3
+            /* "#utility.yul":82:119   */
+          mstore
+            /* "#utility.yul":72:125   */
+          pop
+          pop
+          jump	// out
+            /* "#utility.yul":131:353   */
+        tag_7:
+          0x00
+            /* "#utility.yul":262:264   */
+          0x20
+            /* "#utility.yul":251:260   */
+          dup3
+            /* "#utility.yul":247:265   */
+          add
+            /* "#utility.yul":239:265   */
+          swap1
+          pop
+            /* "#utility.yul":275:346   */
+          tag_15
+            /* "#utility.yul":343:344   */
+          0x00
+            /* "#utility.yul":332:341   */
+          dup4
+            /* "#utility.yul":328:345   */
+          add
+            /* "#utility.yul":319:325   */
+          dup5
+            /* "#utility.yul":275:346   */
+          tag_10
+          jump	// in
+        tag_15:
+            /* "#utility.yul":229:353   */
+          swap3
+          swap2
+          pop
+          pop
+          jump	// out
+            /* "#utility.yul":359:436   */
+        tag_13:
+          0x00
+            /* "#utility.yul":425:430   */
+          dup2
+            /* "#utility.yul":414:430   */
+          swap1
+          pop
+            /* "#utility.yul":404:436   */
+          swap2
+          swap1
+          pop
+          jump	// out
+
+        auxdata: 0xa2646970667358221220847c9788d95894e484e81d4710aff9be15a68f619a87830acd37d4ad818af99764736f6c63782b302e382e342d646576656c6f702e323032312e332e32362b636f6d6d69742e64373164313530352e6d6f64005c
+   }
+
+
+Alternatively, the above output can also be obtained from `Remix <https://remix.ethereum.org/>`_,
+under the option "Compilation Details" after compiling a contract.
+
+Notice that the ``asm`` output starts with the creation / constructor code. The deploy code is
+provided as part of the sub object (in the above example, it is part of the sub-object ``sub_0``).
+The ``auxdata`` field corresponds to the contract :ref:`metadata
+<encoding-of-the-metadata-hash-in-the-bytecode>`.
+
```

Comment:
Can you explain that the comments tell you which source part the code was generated for and that `#utility.yul` is an internally-generated file of utility functions that can be obtained using `--combined-json generated-sources,generated-sources-runtime`?

Reply:
Added this.